youtube_dl/extractor/rtvs.py

```diff
@@ -36,12 +43,66 @@ class RTVSIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, video_id)
+        if url.find('/radio/') != -1:
+            a2 = url.split('/')[-1]
+            a1 = url.split('/')[-2]
+            embed = self._download_webpage(
+                "https://www.rtvs.sk/embed/radio/archive/%s/%s" % (a1, a2),
+                video_id)
+            audio_id = re.search('audio5f.json?id=(?P<id>[^\"]+)', embed)
```

Comment:
Depending on where in the page the target may be, consider `self._html_search_regex()` which unescapes the returned match (eg, if it contains `&amp;` that should be `&`, or just `&#0049;` that should be `1`).